README.md

```diff
@@ -1,26 +1,57 @@
-forc-wallet
-===
+# forc-wallet
 
 A forc plugin for managing Fuel wallets.
 
-## Instructions
+## Quickstart
 
-```shell
-cargo run -- --help
+To install `forc-wallet` you can use `cargo install`:
+
+```sh
+cargo install fuel-wallet
 ```
 
+### Initialize
+
+Before creating accounts and signing transactions with them you need to initialize a new HD wallet. To do so:
+
+```sh
+forc-wallet init
```

Comment:
If `forc` is installed, this could also be:
```sh
forc wallet init
```
However, maybe it's best that we keep these examples using `forc-wallet` in case the user doesn't have `forc` installed for whatever reason? We can always use the `forc wallet` syntax in downstream examples (e.g. in the Sway repo).

Reply:
👍  I did use `forc wallet` in the examples at sway book https://github.com/FuelLabs/sway/pull/2656